client/dashboard/sites/settings-transfer-site/index.tsx

```diff
@@ -34,20 +28,85 @@ export default function SettingsTransferSite( { siteSlug }: { siteSlug: string }
 					title={ __( 'Transfer site' ) }
 					description={ createInterpolateElement(
 						__(
-							'Transfer this site to a new or existing site member with just a few clicks. <learnMoreLink />.'
+							'Transfer this site to a new or existing site member with just a few clicks. <link>Learn more</link>.'
 						),
 						{
-							learnMoreLink: <ExternalLink href="#learn-more">{ __( 'Learn more' ) }</ExternalLink>,
+							// @ts-expect-error children prop is injected by createInterpolateElement
+							link: <ExternalLink href="#learn-more" />,
```

Comment:
This doesn't go anywhere for me.

Reply:
Should be handled by this issue DOTCOM-13301.